.ci/jobs.yml

```diff
@@ -18,4 +18,4 @@ JOB:
   # - x-pack-visualRegression
 
 # `~` is yaml for `null`
-exclude: ~
+exclude: ~
```

Comment:
Please use the `editorconfig` plugin so that little irrelevant changes like this don't occur because we've defined an [.editorconfig](https://github.com/elastic/kibana/blob/master/.editorconfig).

Reply:
Got it, thx!